packages/app/src/__deprecated__/main/main.ts

```diff
@@ -515,3 +518,11 @@ ipcMain.answerRenderer(
 ipcMain.answerRenderer(OutlookAuthActions.CloseWindow, () => {
   outlookAuthWindow?.close()
 })
+
+app.on("second-instance", () => {
+  if (win) {
+    // Focus on the main window if the user tried to open another
+    if (win.isMinimized()) win.restore()
```

Comment:
please, add brackets to if :)

Reply:
those changes wasn't supposed to be there. Going to delete them i a sec : O